packages/scanner/test/scanner/jwtAlgorithmNone.spec.ts

```diff
@@ -0,0 +1,24 @@
+import Check from '../../src/check';
+import rule from '../../src/rules/jwtAlgorithmNone';
+import { scan } from '../util';
+
+describe('JWT Unverified Signature', () => {
+  const check = new Check(rule);
+
+  it('identifies cases where a JWT is encoded and the algorithm is none', async () => {
+    const { findings } = await scan(check, 'JWT.encode_alg_none.appmap.json');
+    expect(findings).toHaveLength(1);
+    expect(findings.map((f) => f.event.id)).toEqual([1]);
```

Comment:
With enumerate scope as false I think this only works because the encode event is the first event in the AppMap.